src/commands/new.ts

```diff
@@ -7,13 +7,16 @@ import { resolve } from 'path';
 
 const { writeFile, readFile } = fPromises;
 const DEFAULT_ASYNCAPI_FILE_NAME = 'asyncapi.yaml';
+const DEFAULT_ASYNCAPI_TEMPLATE = 'simple.yaml';
 
 export default class New extends Command {
   static description = 'creates a new asyncapi file';
 
   static flags = {
     help: flags.help({ char: 'h' }),
     'file-name': flags.string({ char: 'n', description: 'name of the file' }),
+    'use-example': flags.string({ char: 'u', description: 'boolean if they want to explore examples'}),
```

Comment:
Three things here:

1. `boolean if they want to explore examples`. Who are "they"? My recommendation is that you always try to avoid directing to anyone (see other flags) or, if needed, direct your message to the reader using second person singular, i.e., "you(r)(s)". I think this one can become `ask for examples in the interactive terminal`.
2. That said, I'd completely remove this flag and would always show it unless the `example` flag is passed. Just like we do with `port`, for instance.
3. You're not using this flag at all in your code 😄

Reply:
Yeah, thanks, a mistake from me, maybe copy and paste job, but yeah we don't need this as a flag at all.

Reply:
My 2 cents: I think that it should be `show examples in the interactive terminal` or `explore examples...` or something similar, because for me `ask for...` indicates 3rd person, not 2nd (cli user).

Reply:
Ok, nevermind, I see that param is removed 😅